src/main/scala/com/decodified/scalassh/ScpTransferable.scala

```diff
@@ -49,5 +50,5 @@ abstract class ScpTransferable {
   def download(remotePath: String, localPath: String)(implicit l: TransferListener = defaultListener): Try[Unit] =
     fileTransfer(_.download(remotePath, localPath))(l)
 
-  private def defaultListener = new LoggingTransferListener
+  private def defaultListener = new LoggingTransferListener(LoggerFactory.DEFAULT)
```

Comment:
The fact that downgrading sbt caused that scala file to not compile any more is a proof that resolution mechanism has changed between sbt 1.2.8 and 1.3.3

Reply:
Yeah, this is weird.